charts/sophora-indexing-service/templates/_helpers.tpl

```diff
@@ -40,6 +40,7 @@ helm.sh/chart: {{ include "sophora-indexing-service.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+app: {{ .Release.Name }}
```

Comment:
There is already a label with this value: app.kubernetes.io/instance. Why isn’t that label being used instead?